src/systems/filecoin_blockchain/struct/block/_index.md

```diff
@@ -42,6 +42,8 @@ A syntactically valid full block must have:
 
 - all referenced messages syntactically valid,
 - all referenced parent receipts syntactically valid,
+- the sum of the serialized sizes of all messages is no greater than `block.BlockMaxSize`,
+- the sum of the gas limit of all messages is no greater than `block.BlockGasLimit`.
```

Comment:
may want to remind reader to include cron-activated messages (i.e. implicit messages) w gas cost

Reply:
I intend to explicitly exclude the implicit messages, which will have large, fixed gas limits.